### PR TITLE
posit cloud account

### LINKED ISSUE #100: posit cloud guide
provide guidance on how to request posit cloud access and use

### PR DESCRIPTION
**Pull request by shannonpileggi**
Closes #100 

Draft in progress.

### CODE REVIEW

**Comment by cosimameyer** [comment]
Ignore my started review. GitHub inbox led me directly to the review section and I only realized _after_ starting it that it’s still a draft ☺️

**Comment by cosimameyer** [comment]
Just ping me, when you're ready so that I don't miss it ☺️

**Comment by shannonpileggi** [comment]
@cosimameyer this is ready for review! thank you! 🤗

File: content/organization/tech/accounts/index.en.md [hunk 1]
```diff
@@ -115,3 +115,117 @@ Check out the section about [account security](/organization/tech/security/)
...
+
+### Posit Cloud Use
+
+This section is for R-Ladies event organizers requesting access to Posit Cloud.
+
+All correspondence regarding Posit Cloud should be on the `posit-cloud` channel
+in the R-Ladies organizers slack, which includes both requests to access or
+support related questions.
+
+1. Request access to Posit Cloud for your event by submitting the airtable form with
```
Comment (rivaquiroga) [inline content/organization/tech/accounts/index.en.md:151]: What do you think of making explicit how many days in advance people should request access?

File: content/organization/tech/accounts/index.en.md [hunk 2]
```diff
@@ -115,3 +115,117 @@ Check out the section about [account security](/organization/tech/security/)
...
+
+* Name of the R-Ladies chapter hosting the event.
+* Title of the event.
+* Date of the event.
+* Name of the Posit Cloud space administrator (person responsible for customizing the Posit Cloud space).
+* Email address of the Posit Cloud space administrator.
+* Name of the R-Ladies chapter organizer coordinating the event.
+* Email address of the R-Ladies chapter organizer coordinating the event.
+
+An R-Ladies Posit Cloud administrator will review your request as soon as possible. If you do not receive a response withing 7 days you are welcome to flag to the `posit-cloud` channel on the R-Ladies organizers slack.
```
Comment (rivaquiroga) [inline content/organization/tech/accounts/index.en.md:162]: I was able to infer it, but as a non-native English speaker the meaning of "to flag" in this context wasn't very clear to me at first.

File: content/organization/tech/accounts/index.en.md [hunk 3]
```diff
@@ -115,3 +115,117 @@ Check out the section about [account security](/organization/tech/security/)
...
+access to the Posit Cloud account. 
+
+1. Create a new space. Name the space `chapter-title` and assign to r-ladies account.
+
+![](img/image5.png)
+
+2. Add email of primary workshop leader with an Admin role on the space.
+
+![](img/image7.png)
+
```
Comment (rivaquiroga) [inline content/organization/tech/accounts/index.en.md:229]: What we plan to do with the spaces people create after the workshop is held? Delete them after X days?

File: content/organization/tech/accounts/index.en.md (hunk 2 above)
Comment (shannonpileggi) [reply content/organization/tech/accounts/index.en.md:162]: thank you, that is very helpful!

File: content/organization/tech/accounts/index.en.md (hunk 3 above)
Comment (shannonpileggi) [reply content/organization/tech/accounts/index.en.md:229]: excellent point, thank you!

**Pull request merged**